mujoco_warp/_src/collision_driver_test.py

```diff
@@ -101,6 +101,20 @@ class PrimitiveTest(parameterized.TestCase):
           </worldbody>
         </mujoco>
         """,
+    "sphere_cylinder": """
```

Comment:
There is no collision here 😛 

Here, I made you a model that covers all the branches:
```xml
<mujoco>
  <worldbody>
    <body>
      <joint type="slide" axis="1 0 0"/>
      <joint type="slide" axis="0 1 0"/>
      <joint type="slide" axis="0 0 1"/>
      <geom size="0.1" type="sphere"/>
    </body>
    <body>
      <geom size="0.15 0.2" type="cylinder" euler="30 45 0"/>
    </body>
  </worldbody>
  <keyframe>
    <key name="corner" qpos=".33 0 0"/>
    <key name="cap" qpos=".26 -.14 .1"/>
    <key name="side" qpos="0 -.26 0"/>
  </keyframe>
</mujoco>
```